GVFS/GVFS.Common/Paths.Shared.cs

```diff
@@ -68,5 +69,28 @@ public static string ConvertPathToGitFormat(string path)
         {
             return path.Replace(Path.DirectorySeparatorChar, GVFSConstants.GitPathSeparator);
         }
+
+        public static IEnumerable<string> GetFilesRecursive(string rootDirectory)

```

Comment:
I would make this a private method within the repair class. While its behavior is general, we don't need it anywhere else and it's unlikely that we will. And as a private method in that class, we don't have to be super picky about its perf characteristics (e.g. how much overhead is the `yield return` syntax adding?), but I wouldn't this code getting called from a perf critical code path without more analysis.